core/templates/dev/head/app.js

```diff
@@ -102,6 +102,8 @@ oppia.constant('DEFAULT_PROFILE_IMAGE_PATH', '/avatar/user_blue_72px.png');
 oppia.constant('FEEDBACK_POPOVER_PATH',
   '/pages/exploration_player/feedback_popup_container_directive.html');
 
+oppia.constant('LOGOUT_REDIRECT_URL', '/logout?return_url=%2F');
```

Comment:
You can have just `/logout` here since the `/` is default https://github.com/oppia/oppia/blob/e476a91b9f7b34e4adb46a85f08eff62647136a2/core/controllers/base.py#L90

Reply:
Done.